validation/module_schema.yaml

```diff
@@ -113,7 +113,7 @@ parameters:
                                     # Short description (one line)
                                     type: string
                                     required: true
-                                    maxlength: 70
+                                    maxlength: 120
```

Comment:
This is on purpose. Can you instead shorten the ones that are too long?

Reply:
If we have too many offenders to fix immediately we might be better to go as is then follow up on the side gradually ratcheting the max length enforcement down. We should also be periodically reviewing actual usage in QGC (on small screens), a lot of the short descriptions are pretty bad.

Reply:
@bkueng @dagar There are a *lot* of offenders, I started shortening and then realized it was not very easy to do, especially without context on what all parameters do. And yes a lot of the descriptions are pretty bad.

Even with the new maxlength of 120 I needed to shorten a few. Can we gradually work on this afterwards?

Reply:
Alright